### PR TITLE
CHI-2573 Truncate skill name with long length

### PR DESCRIPTION
**Pull request by mythilytm**
## Description
- Addressed this 'bug' - https://tech-matters.slack.com/archives/C035K8919U7/p1711059364850309
- The skill name gets truncated if longer than 30 characters along with a tooltip

### Checklist
- [ ] Corresponding issue has been opened
- [ ] New tests added
- [ ] Feature flags added
- [ ] Strings are localized
- [ ] Tested for chat contacts
- [ ] Tested for call contacts

### Other Related Issues

None

### AFTER YOU MERGE

1. Cut a release tag using the Github workflow. Wait for it to complete and notify in the #aselo-deploys Slack channel.
2. Comment on the ticket with the release tag version AND any additional instructions required to configure an environment to test the changes.
3. Only then move the ticket into the QA column in JIRA

You are responsible for ensuring the above steps are completed. If you move a ticket into QA without advising what version to test, the QA team will assume the latest tag has the changes. If it does not, the following confusion is on you! :-P

### CODE REVIEW

File: plugin-hrm-form/src/components/teamsView/SkillChip.tsx [hunk 1]
```diff
@@ -40,7 +41,13 @@ const SkillChip: React.FC<Props> = ({ skill, skillType }) => {
       {skillType === 'active' && <SmallCheckIcon htmlColor={fontColor} />}
       {skillType === 'disabled' && <SmallDisabledIcon htmlColor={fontColor} />}
       <ChipText bold color={fontColor}>
-        {skill}
+        {skill.length > 30 ? (
+          <Tooltip title={skill}>
+            <span>{`${skill.substring(0, 30)}...`}</span>
```
Comment (paulschreiber) [inline plugin-hrm-form/src/components/teamsView/SkillChip.tsx:46]:
```suggestion
            <span>{`${skill.substring(0, 30)}…`}</span>
```

Comment (paulschreiber) [reply plugin-hrm-form/src/components/teamsView/SkillChip.tsx:46]: `option-;` will give you a true ellipsis.

Comment (mythilytm) [reply plugin-hrm-form/src/components/teamsView/SkillChip.tsx:46]: Neat. This is good to know. What is the difference?

Comment (mythilytm) [reply plugin-hrm-form/src/components/teamsView/SkillChip.tsx:46]: Neat, that is good to know. How can I get this to work in the ide

**Review by murilovmachado — APPROVED** [review]
LGTM

File: plugin-hrm-form/src/components/teamsView/SkillChip.tsx (hunk 1 above)
Comment (mythilytm) [reply plugin-hrm-form/src/components/teamsView/SkillChip.tsx]: thanks. this is good to know

**Pull request merged**